### ISSUE TITLE
HCB and HCW avatar bug

### ISSUE THREAD

**Issue by rengare** [SAP/fundamental-styles#1192]
**Describe the bug**
The avatar component displays the wrong colour according to the fiori3 specification.
The problem is that variables are for normal mode and HCB are different. The colour should be changed on variables side under the hood. 

![image](https://user-images.githubusercontent.com/10849982/86021587-866df500-ba29-11ea-880e-564bae95ed2c.png)

**Comment by rengare** [comment]
Check #1249
